### DISCUSSION TITLE
drop `selinux` option

### DISCUSSION THREAD

**Discussion by haircommander** [cri-o/cri-o#6511] [Ideas]
Currently, the `selinux` option in the cri-o config automatically is set to whether SELinux is enabled on the host. While it's possible to turn of selinux while still running it on the host, I find this to be a strange case. I imagine admins either want selinux, or don't, and so the configuration option could just be replaced by automatically detecting if it's enabled.

What do folks think? Can you think of a case where we'd want the selinux option?

**Comment by haircommander** [comment]
cc @cri-o/cri-o-maintainers @rhatdan

**Comment by saschagrunert** [comment]
I'm in favor of dropping the configuration and documenting that SELinux can be managed through `setenforce` automatically.

**Comment by jrvaldes** [comment]
@haircommander @saschagrunert  is there a GH issue or Jira card to track this work?

**Reply by haircommander** [reply]
not yet, we could create an issue, or just have a PR be opened :)

**Reply by haircommander** [reply]
do you have interest in working on this?

**Reply by jrvaldes** [reply]
I'm trying to correlate the dependencies with https://github.com/cri-o/cri-o/issues/6499
